capa/main.py

```diff
@@ -863,7 +863,12 @@ def main(argv=None):
         # so we can fairly quickly determine if the given PE file has "pure" file-scope rules
         # that indicate a limitation (like "file is packed based on section names")
         # and avoid doing a full code analysis on difficult/impossible binaries.
-        file_extractor = capa.features.extractors.pefile.PefileFeatureExtractor(args.sample)
+        try:
+            from pefile import PEFormatError
+            file_extractor = capa.features.extractors.pefile.PefileFeatureExtractor(args.sample)
```

Comment:
this addition mitigated the issue already, here we display a nicer error message